opencensus/trace/span.py

```diff
@@ -346,6 +346,30 @@ def add_link(self, link):
             raise TypeError("Type Error: received {}, but requires Link.".
                             format(type(link).__name__))
 
+    def set_status(self, code, message='', details=None):
```

Comment:
Please take a look at https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/status.py, we might need to consolidate them.

Reply:
Do you think we should change the method to receive a `Status`? I followed the guidelines of method signature.

Or do you think we should eliminate the `Status` class?

Reply:
Personally I want to omit `Status` class since it feels more Pythonic to me. We've also tried something similar in #602 to avoid introducing too many plain classes.

@c24t what do you think?

Reply:
Looking at [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#set-status), it seems the `set_status` API should only take one argument, e.g. `Span.set_status(status: Status)`.
```
The Span interface MUST provide:
An API to set the Status where the new status is the only argument. This SHOULD be called SetStatus.
```

Reply:
I do think the `Status` encapsulates some logic and I'm fine with using it.

It might be interesting to add some *default* values to the `Status` class, line `Ok`, `Unknown` and etc...

What do you think?

Reply:
> Looking at the spec, it seems the set_status API should only take one argument, e.g. Span.set_status(status: Status).

How could I miss that... changing now.

Reply:
I resolved this, please take a look.

Reply:
> It might be interesting to add some default values to the Status class, line Ok, Unknown and etc...
> What do you think?

I agree with you.